101-bundle-json.md

```diff
@@ -2,6 +2,10 @@
 
 This section describes the format and function of the `bundle.json` document.
 
+The `bundle.json` file is a representation of bundle metadata. It MUST be represented as [Canonical JSON](http://wiki.laptop.org/go/Canonical_JSON). While Canonical JSON is parseable by any JSON parser, its serialized form is consistent. This is a necessity when comparing two textual representations of the same data (such as when hashing).
```

Comment:
Should we add appropriate verbiage around whether or not tooling should error if the bundle.json is not represented in the correct format?

Reply:
This part is tricky. I haven't seen any Canonical JSON parsers that actually verify that the content it parses is canonical. But we could certainly add a MAY line in here.